src/main/java/homework/Fruit.java

```diff
@@ -1,24 +1,19 @@
-package homework.home10012019;
```

Comment:
I many times said:"In homework I must look only on your homework tasks". It will be changes only in two files. 
Look at the "green 100%" homework other students